daru.gemspec

```diff
@@ -69,7 +69,7 @@ EOF
   spec.add_development_dependency 'dbi'
   spec.add_development_dependency 'activerecord', '~> 4.0'
   spec.add_development_dependency 'sqlite3'
-  spec.add_development_dependency 'rubocop', '>= 0.40.0'
+  spec.add_development_dependency 'rubocop', '~> 0.49.0'
```

Comment:
@zverok it's a bit of offtopic but I always struggled to understand why could it be useful to have such development dependencies in the Gemspec for projects which use Bundler. The only valid usecase for me was to download the gem and use it as "source package" for development. Do you have a strong opinion about Gemfile vs. Gemspec for development dependencies? Thank you!